src/Symfony/Bundle/DebugBundle/DependencyInjection/Configuration.php

```diff
@@ -50,6 +50,12 @@ public function getConfigTreeBuilder()
                     ->example('php://stderr, or tcp://%env(VAR_DUMPER_SERVER)% when using the "server:dump" command')
                     ->defaultNull()
                 ->end()
+                ->enumNode('html_dumper_theme')
```

Comment:
I'd just call this "theme"

Reply:
I did it because it only changes the html render, not the console one. But I'll make the change.

Thanks!